web/massastation/src/custom/api/usePost.ts

```diff
@@ -28,9 +28,10 @@ export function usePost<TBody, TResponse = null>(
     mutationFn: async ({ params, payload }) => {
       const queryParams = new URLSearchParams(params).toString();
       if (queryParams) {
-        url.concat(`?${queryParams}`);
+        url = `${url}?${queryParams}`;
```

Comment:
`.concat` is better than doing this 🤔

Reply:
it didn't work with xoncat

Reply:
this does not work but if you want we can do this : 
```
url = url.concat(`?${queryParams}`);
```

Reply:
I think that `    url = `${url}?${queryParams}`;` is easier to read that concat

Reply:
> this does not work but if you want we can do this :
> 
> ```
> url = url.concat(`?${queryParams}`);
> ```

Thank you! Better on this way